nncf/torch/nncf_network.py

```diff
@@ -142,11 +142,6 @@ def __hash__(self):
         return hash(str(self))
 
 
-class ExtraCompressionModuleType(Enum):
```

Comment:
Could you explain the idea of refactoring relative to `ExtraCompressionModuleType`? What kind of problem did you try to solve?

Reply:
I did it because I need `ExtraCompressionModuleType` in `nncf/torch/graph/transformations/commands.py` file and it caused circular import error. I decided to moved this enum to the commands file then